client/src/spotify/creds.js

```diff
@@ -0,0 +1,9 @@
+const Credentials = () => {
+
+  return {
+    ClientId: 'a1516982e6044f1cac493ab1c8492818',
+    ClientSecret: 'a3be153a02124321bdbd647aa70732b4'
```

Comment:
we should not commit this to the repo. we should add something like [dotenv](https://github.com/motdotla/dotenv) to read from an environment file